RELEASE_NOTES.md

```diff
@@ -1,3 +1,15 @@
+## Cookstyle 6.8
+
+### Improvements for Chef Infra Client 16.2
+
+Chef Infra Client 16.2 contains additional changes to how resource names are specified in order to avoid edge cases in resource naming. When setting the name of a custom resource `provides` should now be used instead of `resource_name`. If writing cookbooks that support Chef Infra Client < 16 you'll want to use both `provides` and `resource_name` to support all all releases.
```

Comment:
```suggestion
Chef Infra Client 16.2 contains additional changes to how resource names are specified in order to avoid edge cases in resource naming. When setting the name of a custom resource `provides` should now be used instead of `resource_name`. If writing cookbooks that support Chef Infra Client < 16 you'll want to use both `provides` and `resource_name` to support all releases.
```